Essentials/pom.xml

```diff
@@ -95,12 +107,6 @@
             <artifactId>ReflectionProvider</artifactId>
             <version>2.16.0</version>
             <scope>compile</scope>
-            <exclusions>
-                <exclusion>
-                    <groupId>org.bukkit</groupId>
-                    <artifactId>craftbukkit</artifactId>
-                </exclusion>
-            </exclusions>
```

Comment:
What actual effect does this have?

Reply:
That module doesn't have craftbukkit as a dependency, so excluding it does absolutely nothing - that's why I have removed it.